moped-database/metadata/tables.yaml

```diff
@@ -2807,6 +2807,51 @@
 - table:
     schema: public
     name: moped_project
+  array_relationships:
+  - name: moped_proj_dates
+    using:
+      foreign_key_constraint_on:
+        column: project_id
+        table:
+          schema: public
+          name: moped_proj_dates
+  - name: moped_proj_entities
+    using:
+      foreign_key_constraint_on:
+        column: project_id
+        table:
+          schema: public
+          name: moped_proj_entities
+  - name: moped_proj_financials
+    using:
+      manual_configuration:
+        remote_table:
+          schema: public
+          name: moped_proj_financials
+        column_mapping:
+          project_id: project_id
+  - name: moped_proj_phases
+    using:
+      foreign_key_constraint_on:
+        column: project_id
+        table:
+          schema: public
+          name: moped_proj_phases
+  - name: moped_proj_status_history
+    using:
+      foreign_key_constraint_on:
+        column: project_id
+        table:
+          schema: public
+          name: moped_proj_status_history
+  - name: moped_proj_timeline
+    using:
+      manual_configuration:
+        remote_table:
+          schema: public
+          name: moped_proj_timeline
+        column_mapping:
+          project_id: project_id
```

Comment:
This is great! 💟  I can see the relationships working in the local database:

![2020-12-09_16-30-21](https://user-images.githubusercontent.com/5282430/101696518-d7076680-3a3b-11eb-95e8-4b05ede6ba4b.png)